src/main/java/Event.java

```diff
@@ -0,0 +1,19 @@
+public class Event extends Task {
+
+    protected String at;
+
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+    }
+
+    @Override
+    public String getType() {
```

Comment:
You might want to change the name of this method to getTaskType for better understandability of the code from 3rd party!